frontend/javascripts/router.js

```diff
@@ -449,7 +445,6 @@ class ReactRouter extends React.Component<Props> {
                   return <FinishResetPasswordView resetToken={params.token} />;
                 }}
               />
-              <Route path="/spotlight" component={SpotlightView} />
```

Comment:
so, this route is not supported anymore, correct?

Reply:
Yeah. I don't think it was linked anywhere anymore.